packages/components/src/utils/useMediaQueries.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useEffect, useState, ReactNode, useMemo } from "react"
+import { useTheme } from "@kaizen/design-tokens"
```

Comment:
Should this point to the one in KaizenProvider instead?

Reply:
Using KaizenProvider version breaks tests... ticke to investigate another time.